examples/infrastructure/vpc.yaml

```diff
@@ -202,6 +201,55 @@ Resources:
       RouteTableId: !Ref PrivateRouteTable2
       SubnetId: !Ref PrivateSubnet2
 
+  PublicLoadBalancerSG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Access to the public facing load balancer
+      VpcId: !Ref VPC
+      SecurityGroupIngress:
+          - CidrIp: 0.0.0.0/0
+            IpProtocol: -1
+
+  PublicLoadBalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Scheme: internet-facing
+      LoadBalancerAttributes:
+        - Key: idle_timeout.timeout_seconds
+          Value: '30'
+      Subnets:
+        - !Ref PublicSubnet1
+        - !Ref PublicSubnet2
+      SecurityGroups: [!Ref 'PublicLoadBalancerSG']
+
+  # Initial placeholder target group to drop traffic; actual target group
+  # creation is deferred to a later application stack
+  PlaceholderTargetGroupPublic:
```

Comment:
Instead of this, we could create the load balancer itself in the application stack.

Reply:
Just a perf optimization to create it with the VPC since this gets torn down less frequently than the app stack. I had it there originally, so let me know if that's what you really prefer.

Reply:
I think application stack makes the most sense, even if it takes a little bit longer. Don't want this to seem like a recommendation on how to split up infrastructure stacks (hotly debated either way).

Reply:
We keep it separate in the voteapp because we got so tired of teardown time on redeploys, but yeah, sure, I'll move to the app stack.

Reply:
(right after lunch) :)